cas-server-core/src/main/java/org/jasig/cas/ticket/TicketGrantingTicketImpl.java

```diff
@@ -68,7 +68,12 @@
     @Lob
     @Column(name="SERVICES_GRANTED_ACCESS_TO", nullable=false)
     private final HashMap<String,Service> services = new HashMap<String, Service>();
-    
+
+    /** Service that produced a proxy-granting ticket. */
+    @Column(name="PROXIED_BY", nullable=true)
+    private String proxiedBy;
```

Comment:
It's not set in all cases -- there's a no-arg constructor. Also, it's consistent with other non-collection fields.